mesa/experimental/jupyter_viz.py

```diff
@@ -300,3 +372,22 @@ def function(model):
         solara.Markdown(renderer(model))
 
     return function
+
+
+def get_initial_grid_layout(layout_types):
+    grid_lay = []
+    cord_y = 0
```

Comment:
@ankitk50 great! one minor edit on this and I am about to do a pull request for the history file so we can do a release @jackiekazil 

could you change this to `y_coord` this will make it consistent with other mesa variables for y coordinates following principle of least surprise.